app/grants/management/commands/estimate_clr.py

```diff
@@ -49,14 +49,14 @@ def handle(self, *args, **options):
 
         if active_clr_rounds:
             for clr_round in active_clr_rounds:
-                print(f"CALCULATING CLR estimates for ROUND: {clr_round.round_num}")
-                predict_clr(
+                print(f"CALCULATING CLR estimates for ROUND: {clr_round.round_num} {clr_round.sub_round_slug}")
+                process_predict_clr(
                     save_to_db=True,
                     from_date=timezone.now(),
                     clr_round=clr_round,
                     network=network
                 )
-                print(f"finished CLR estimates for {clr_round.round_num}")
+                print(f"finished CLR estimates for {clr_round.round_num} {clr_round.sub_round_slug}")
 
                 # TOTAL GRANT
                 # grants = Grant.objects.filter(network=network, hidden=False, active=True, link_to_new_grant=None)
```

Comment:
^ I believe we'd have to move this commented out code as well into process_predict_clr
this is used for debugging

Reply:
oh right!